### PR TITLE
Create PrometheusHA workbook

### PR DESCRIPTION
**Pull request by rsponholtz**
Added v2.0 workbook

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook [hunk 1]
```diff
@@ -0,0 +1,1215 @@
...
+            "id": "92e2c347-22df-4b92-9de7-54a4088c3929",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_provider_instance",
+            "type": 1,
+            "value": "HACLUSTER"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.hanaonazure/sapmonitors"
```
Comment (archanak-msft) [inline Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook:45]: This needs to be changed to the one we have in v2.

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook [hunk 2]
```diff
@@ -0,0 +1,1215 @@
...
+                  "resultValType": "static",
+                  "resultVal": "0"
+                }
+              }
+            ]
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.hanaonazure/sapmonitors"
```
Comment (archanak-msft) [inline Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook:93]: change this as well

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook [hunk 3]
```diff
@@ -0,0 +1,1215 @@
...
+                  "isGlobal": true,
+                  "timeContext": {
+                    "durationMs": 86400000
+                  },
+                  "value": "hacluster"
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.hanaonazure/sapmonitors"
```
Comment (archanak-msft) [inline Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook:1113]: thisn as well

**Review by archanak-msft — CHANGES_REQUESTED** [review]

**Comment by archanak-msft** [comment]
@Hiren ***@***.***> and @Dhruv ***@***.***> - can you please help with this?

-Archana

**Comment by dhruv** [comment]
You've got the wrong @dhruv again.

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook (hunk 1 above)
Comment (rsponholtz) [reply Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook]: This is corrected.

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook (hunk 2 above)
Comment (rsponholtz) [reply Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook]: This is corrected.

File: Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook (hunk 3 above)
Comment (rsponholtz) [reply Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook]: This is corrected.

**Review by hirenchalodiya1 — APPROVED** [review]

**Pull request merged**